ksmt-bitwuzla/src/main/kotlin/org/ksmt/solver/bitwuzla/KBitwuzlaExprConverter.kt

```diff
@@ -735,8 +735,7 @@ open class KBitwuzlaExprConverter(
     }
 
     private inner class BoolToBv1AdapterExpr(val arg: KExpr<KBoolSort>) : KExpr<KBv1Sort>(ctx) {
-        override val sort: KBv1Sort
-            get() = ctx.bv1Sort
+        override val sort: KBv1Sort = ctx.bv1Sort
```

Comment:
I thought we didn't want to store sorts in the objects due to memory limitations. Why is it not applicable here?

Reply:
Since any expr type access stores the type in the cache we will store the types for all expressions anyway.